### PR TITLE
Update python ectoken

### PR DESCRIPTION
**Pull request by R4wm**
[replace OpenSSL.rand for os.urandom](https://www.pyopenssl.org/en/stable/changelog.html?highlight=urandom#id35)
port ectoken to py3

### CODE REVIEW

**Comment by R4wm** [comment]
example use: https://pastebin.com/FrpMAKgF

File: python-ectoken/ectoken3.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```
Comment (airencracken) [inline python-ectoken/ectoken3.py]: Prefer `#!/usr/bin/env python3`

File: python-ectoken/ectoken3.py [hunk 2]
```diff
@@ -42,9 +42,9 @@
 import hashlib
 
 import OpenSSL
```
Comment (airencracken) [inline python-ectoken/ectoken3.py]: No longer needed?

File: python-ectoken/requirements.txt [hunk 3]
```diff
@@ -13,6 +13,6 @@
 # * limitations under the License.
 # */
 
-pyOpenSSL>=0.15.1
+pyOpenSSL>=19.0.0
```
Comment (airencracken) [inline python-ectoken/requirements.txt]: No longer necessary?

File: python-ectoken/ectoken3.py [hunk 4]
```diff
@@ -214,13 +206,13 @@ def main(argv):
         except Exception as e:
             if l_args.verbose:
                 print('| Failed to decrypt v3 token trying to decrypt as v1/2 token')
-                print('| Error detail: type: %s error: %s, doc: %s, message: %s'% (type(e), e, e.__doc__, e.message))
```
Comment (R4wm) [inline python-ectoken/ectoken3.py]: no e.message here, so removed

File: python-ectoken/ectoken3.py (hunk 2 above)
Comment (R4wm) [reply python-ectoken/ectoken3.py]: removed all unused imports, ty

File: python-ectoken/requirements.txt (hunk 3 above)
Comment (R4wm) [reply python-ectoken/requirements.txt]: removed pyOpenSSL in requirements and comments , ty

**Comment by airencracken** [comment]
lgtm as long as most of the cross language tests pass. Need to get that passing in CI/CD so running them manually isn't a thing. Get @jcline to approve.

**Comment by R4wm** [comment]
Just a couple manual interoperability tests: 

`(virt-ectoken) ᚱ@c++-ectoken $ echo $PWD && ../py3-ectoken/ectoken3.py -k somekey -t $(./ectoken3 -e somekey sometoken) -d
/home/rmintz/git/r4wm/ectoken/c++-ectoken
sometoken
(virt-ectoken) ᚱ@c++-ectoken $ `

`(virt-ectoken) ᚱ@py3-ectoken $ ./ectoken3.py -k somekey -t $($GOPATH/src/github.com/VerizonDigital/ectoken/go-ectoken/ectoken3 somekey sometoken) -d
sometoken
(virt-ectoken) ᚱ@py3-ectoken $ `

**Pull request merged**